### PR TITLE
manage albums

### LINKED ISSUE #20: managing albums basics
This means

- [x] create album
- [x] delete album 
- [x] rename album
- [x] assign album to a collection (needs #21 to be done)
- [x] move an album from a collection to another (needs #21 to be done)
- [x] assign a photo to an album (needs #22 to be done)
- [x] documentation
- [x] tests

### PR DESCRIPTION
**Pull request by Sulfiore**
closes #20

### CODE REVIEW

File: src/collection.rs [hunk 1]
```diff
@@ -66,6 +66,16 @@ impl Collection
 		}
 	}
 
+	pub fn set_id (&mut self, id: u32)
+	{
+		self.id = id;
+	}
+
+	pub fn id (&self) -> u32
```
Comment (ADBeveridge) [inline src/collection.rs:74]: This should probably be named `get_id`.

File: src/photo.rs [hunk 2]
```diff
@@ -88,6 +88,19 @@ impl Photo
 	}
 }
 
+impl Photo
+{
+	pub fn id(&self) -> u32
```
Comment (ADBeveridge) [inline src/photo.rs:93]: This also, see above, might be just as well to call this `get_id`

File: src/album.rs [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+// Getters and setters
+impl Album
+{
+	pub fn set_id(&mut self, id: u32)
+	{
+		self.id = id;
+	}
+
+	pub fn id(&self) -> u32
```
Comment (ADBeveridge) [inline src/album.rs:85]: `get_id`()

**Review by ADBeveridge — CHANGES_REQUESTED** [review]

File: src/collection.rs (hunk 1 above)
Comment (Sulfiore) [reply src/collection.rs:74]: It shouldn't if we respect the rust api guidelines 

https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

File: src/photo.rs (hunk 2 above)
Comment (Sulfiore) [reply src/photo.rs:93]: Cf. Previous answer

File: src/album.rs (hunk 3 above)
Comment (Sulfiore) [reply src/album.rs:85]: Cf. Previous answer

File: src/collection.rs (hunk 1 above)
Comment (ADBeveridge) [reply src/collection.rs:74]: ok

**Review by ADBeveridge — APPROVED** [review]

**Pull request merged**